app/frontend/src/pages/HomePage/HomePage.js

```diff
@@ -8,19 +8,19 @@ export default function HomePage() {
 
     const groupData = [
         {
-            image: `https://primefaces.org/cdn/primereact/images/product/game-controller.jpg}`,
+            image: `https://i.hizliresim.com/o2d7842.jpg`,
             header: '"Looking for a co-op adventure in Hyrule!"',
             text: "Need a skilled squad for high-intensity chicken dinners! Who's in for some PUBG action? Let's conquer the battleground together! 🍗🔫",
             players: '4/5'
         },
         {
-            image: `https://primefaces.org/cdn/primereact/images/product/game-controller.jpg}`,
+            image: `https://i.hizliresim.com/oo1m2tv.jpg`,
             header: '"Searching for squadmates for some intense battle royale action!"',
             text: "Building forts and taking names! Searching for fellow Fortnite warriors to join my squad. Let's get those Victory Royales! 🏰👑",
             players: '4/5'
         },
         {
-            image: `https://primefaces.org/cdn/primereact/images/product/game-controller.jpg}`,
+            image: `https://i.hizliresim.com/glo9hgx.jpg`,
```

Comment:
Put our images into the images folder and use them here, so that we will be prepared to the scenarios in which link are broken or there's a problem with the connection.

Reply:
I have made the requested changes, could you check again?